ext/civicrm_admin_ui/managed/SavedSearch_Administer_MembershipStatusRules.mgd.php

```diff
@@ -0,0 +1,236 @@
+        'type' => 'table',
+        'settings' => [
+          'description' => NULL,
+          'sort' => [],
+          'limit' => 50,
+          'pager' => [],
+          'placeholder' => 5,
+          'columns' => [
+            [
+              'type' => 'field',
+              'key' => 'label',
+              'dataType' => 'String',
+              'label' => E::ts('Status'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_active',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Enabled'),
+              'sortable' => TRUE,
+              'cssRules' => [],
+            ],
+            [
+              'type' => 'field',
+              'key' => 'start_event:label',
+              'dataType' => 'String',
+              'label' => E::ts('Start Event'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'start_event_adjust_interval',
+              'dataType' => 'Integer',
+              'label' => E::ts('Start Adjustment'),
+              'sortable' => TRUE,
+              'rewrite' => '[start_event_adjust_interval] [start_event_adjust_unit:label]',
+            ],
+            [
+              'type' => 'field',
+              'key' => 'end_event:label',
+              'dataType' => 'String',
+              'label' => E::ts('End Event'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'end_event_adjust_interval',
+              'dataType' => 'Integer',
+              'label' => E::ts('End Adjustment'),
+              'sortable' => TRUE,
+              'rewrite' => '[end_event_adjust_interval] [end_event_adjust_unit:label]',
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_current_member',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Current Member?'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_admin',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Administrator Only?'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_reserved',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Is Reserved'),
+              'sortable' => TRUE,
+            ],
+            [
+              'size' => 'btn-xs',
+              'links' => [
+                [
+                  'entity' => '',
+                  'action' => '',
+                  'join' => '',
+                  'target' => '',
+                  'icon' => 'fa-pencil',
+                  'text' => E::ts('Edit'),
+                  'style' => 'default',
+                  'path' => 'civicrm/admin/member/membershipStatus?action=update&id=[id]&reset=1',
```

Comment:
@mattwire also this should be changed out to use entity+action instead of hardcoded path.